build/constants.yaml

```diff
@@ -160,29 +160,29 @@
     python311-version: 3.11.8
     python312-version: 3.12.2
     runtime-versions:
-      - 3.7-debian-bullseye
-      - 3.7-debian-buster
-      - 3.8-debian-bookworm
-      - 3.8-debian-bullseye
-      - 3.8-debian-buster
-      - 3.9-debian-bookworm
-      - 3.9-debian-bullseye
-      - 3.9-debian-buster
-      - 3.10-debian-bookworm
-      - 3.10-debian-bullseye
-      - 3.10-debian-buster
-      - 3.11-debian-bookworm
-      - 3.11-debian-bullseye
-      - 3.12-debian-bookworm
-      - 3.12-debian-bullseye
-      - dynamic-debian-buster
+    - 3.7-debian-bullseye
+    - 3.7-debian-buster
+    - 3.8-debian-bookworm
+    - 3.8-debian-bullseye
+    - 3.8-debian-buster
+    - 3.9-debian-bookworm
+    - 3.9-debian-bullseye
+    - 3.9-debian-buster
+    - 3.10-debian-bookworm
+    - 3.10-debian-bullseye
+    - 3.10-debian-buster
+    - 3.11-debian-bookworm
+    - 3.11-debian-bullseye
+    - 3.12-debian-bookworm
+    - 3.12-debian-bullseye
+    - dynamic-debian-buster
```

Comment:
any idea on why this automation keeps changing the formatting with each PR?

Reply:
It's like automation workflow indent the content of the file.
From next PR, i think it wont be like this.